theseus_gui/src/pages/project/Version.vue

```diff
@@ -0,0 +1,350 @@
+        <div class="metadata">
+          <div class="metadata-item">
+            <span class="metadata-label">Release Channel</span>
+            <span class="metadata-value"
+              ><Badge
+                :color="releaseColor(version.version_type)"
+                :type="
+                  version.version_type.charAt(0).toUpperCase() + version.version_type.slice(1)
+                "
+            /></span>
+          </div>
+          <div class="metadata-item">
+            <span class="metadata-label">Version Number</span>
+            <span class="metadata-value">{{ version.version_number }}</span>
+          </div>
+          <div class="metadata-item">
+            <span class="metadata-label">Loaders</span>
+            <span class="metadata-value">{{
+              version.loaders
+                .map((loader) => loader.charAt(0).toUpperCase() + loader.slice(1))
+                .join(', ')
+            }}</span>
+          </div>
+          <div class="metadata-item">
+            <span class="metadata-label">Game Versions</span>
+            <span class="metadata-value"> {{ version.game_versions.join(', ') }} </span>
+          </div>
+          <div class="metadata-item">
+            <span class="metadata-label">Downloads</span>
+            <span class="metadata-value">{{ version.downloads }}</span>
+          </div>
+          <div class="metadata-item">
+            <span class="metadata-label">Publication Date</span>
+            <span class="metadata-value">
+              {{
+                new Date(version.date_published).toLocaleString('en-US', {
+                  month: 'long',
+                  day: 'numeric',
+                  year: 'numeric',
+                })
+              }}
+              at
+              {{
+                new Date(version.date_published).toLocaleString('en-US', {
+                  hour: 'numeric',
+                  minute: 'numeric',
+                  second: 'numeric',
+                  hour12: true,
+                })
+              }}
+            </span>
+          </div>
+          <div v-if="author" class="metadata-item">
+            <span class="metadata-label">Author</span>
+            <a
+              :href="`https://modrinth.com/user/${author.user.username}`"
+              rel="external"
+              target="_blank"
+              class="metadata-value btn author"
+            >
+              <Avatar size="sm" :src="author.user.avatar_url" circle />
+              <span>
+                <strong>
+                  {{ author.user.username }}
+                </strong>
+                <br />
+                {{ author.role }}
+              </span>
+            </a>
+          </div>
+          <div class="metadata-item">
+            <span class="metadata-label">Version ID</span>
+            <span class="metadata-value">{{ version.downloads }}</span>
+          </div>
+        </div>
+      </Card>
+    </div>
+  </div>
+</template>
+
+<script setup>
```

Comment:
also remember to use `ref` and `shallowRef` here for things that are passed to the dom